demos/admin_panel_bot/screens.py

```diff
@@ -0,0 +1,105 @@
+"""The module contains the screens the bot consists of."""
+
+from hammett.conf import settings
+from hammett.core import Button, Screen
+from hammett.core.constants import SourcesTypes
+from hammett.core.handlers import register_button_handler
+from hammett.core.hider import ONLY_FOR_ADMIN, Hider
+from hammett.core.mixins import StartMixin
+
+ADMIN_PANEL_SCREEN_DESCRIPTION = (
+    "This is <b>AdminPanelScreen</b>. You've managed to move here, "
+    "because you could see the button that moves to this screen.\n"
+    "\n"
+    "Now remove yourself from the admin group to see what will change."
```

Comment:
```
This is the Admin Panel screen. You’ve made it here because you saw the button that takes you to this screen. Now, go ahead and remove yourself from the admin group to see what changes.
```